pkgs/by-name/gv/gvmd/package.nix

```
@@ -0,0 +1,70 @@
+{ lib, stdenv, fetchFromGitHub, cmake, gvm-libs, pkg-config, gnutls, glib
+, doxygen, libbsd, libical, gpgme, util-linux, libselinux, libsepol, libidn2
+, graphviz, icu, p11-kit, pcre2, libxslt, libtasn1, installShellFiles, xmltoman
+, postgresql, }:
+
+stdenv.mkDerivation rec {
+  pname = "gvmd";
+  version = "23.4.0";
+
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "gvmd";
+    rev = "v${version}";
+    hash = "sha256-Bxq6o98viMMwt2eoiZgPDTYqD/z6jRWYLgH/ZJvkpV4=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config doxygen xmltoman installShellFiles ];
+
+  buildInputs = [
+    libxslt
+    postgresql
+    pcre2
+    libbsd
+    libidn2
+    gvm-libs
+    graphviz
+    gnutls
+    glib
+    libtasn1
+    icu
+    libical
+    gpgme
+    glib
+    libsepol
+    libselinux
+    util-linux
+    p11-kit
+    postgresql
+    gpgme
+  ];
+
+  strictDeps = true;
+
+  prePatch = ''
+    substituteInPlace src/sql_pg.c --replace-warn "#include <postgresql/libpq-fe.h>" "#include <${postgresql}/include/libpq-fe.h>";
```

```suggestion
    substituteInPlace src/sql_pg.c \
      --replace-warn "#include <postgresql/libpq-fe.h>" "#include <${postgresql}/include/libpq-fe.h>"
```
Breaking line because it's a long line. And it also splits the file from the what is being replaced.